api-reference/auth/desktop-sdks/web.mdx

```diff
@@ -178,26 +223,149 @@ const userInfo = connect({
 
 <Note>`web3.eth.getAccounts` will also initiate login automatically if user isn't already logged in.</Note>
 
+#### Log in with a Specific Social Platform
+
+You can configure the login flow to bypass the login modal entirely, prompting the user to log in directly with a specific social platform.
+
+The following example demonstrates how to prompt the user to log in with their Google account, skipping the login modal.
+
+```tsx App.tsx
+  // Handle user login
+  const handleLogin = async () => {
+    if (!connected) {
+      await connect({
+        socialType: "google",
+      });
+    }
+  };
+```
+
 ***
 
+### Verify the User's Login Status
+
+The `useConnect()` hook provides several functions to check the user's connection status:
+
+- `connected` — Returns a boolean value indicating whether the user is logged in (`true`) or not (`false`). This is useful for managing login logic.
+
+- `connectionStatus` — Returns a string representing the current stage of the user's flow:
+  - `loading`
+  - `connecting`
+  - `connected`
+  - `disconnected`
+
+  You can use `connectionStatus` to manage login logic or to display relevant status information in the UI.
+
+### Handling User Information
+
+Once the user is logged in, the `userInfo` object from the `useAuthCore()` hook becomes available. 
+
+This object provides a wealth of information that can enhance the user experience (UX). 
+
+For example, you can use it to display the user's `name` and `avatar`, or access security details such as `has_set_master_password`. This allows you to implement logic that prompts the user to set a master password if they haven’t already, thereby improving security.
+
+<Accordion title="Available Fields in `userInfo`">
```

Comment:
This is in an accordion, so it looks clean in the actual page.

Reply:
Also if you assign the result of `connect`, it'll yield the same object.